### DISCUSSION TITLE
Why don't I have achievements🥲

### DISCUSSION THREAD

**Discussion by YouthguyZ** [Schweinepriester/github-profile-achievements#67] [Q&A] [answered]
oh bro help me 😥

**Comment by quicksandznzn** [comment]
balabala~

**Comment by melvincwng** [comment, marked as answer]
@YouthguyZ, 欢迎来到GitHub 👋

To get achievements, just follow the required steps in the main page of this particular code repository.

In short, make some PRs, merge a PR without review and get 'marked as answer' for 2 discussions to unlock those badges.

**Reply by melvincwng** [reply]
@YouthguyZ, 如果此回复有帮助，请随时在 GitHub 上关注我并“标记为答案”。 谢谢。

**Reply by YouthguyZ** [reply]
thanks man i will try it🤔

**Reply by melvincwng** [reply]
@YouthguyZ, pleasure to help 😄

**Reply by YouthguyZ** [reply]
好兄弟 你是华人吗？
